src/main/java/gov/cabinetoffice/gap/adminbackend/controllers/SubmissionsController.java

```diff
@@ -51,7 +51,7 @@ public ResponseEntity<InputStreamResource> exportSpotlightChecks(@PathVariable I
         long start = System.currentTimeMillis();
 
         final ByteArrayOutputStream stream = submissionsService.exportSpotlightChecks(applicationId);
-        final String exportFileName = submissionsService.generateExportFileName(applicationId);
+        final String exportFileName = "required_checks.zip";
```

Comment:
Could you make this a constant please Rachel? Will stop sonar from complaining about it when we try to commit code.